src/System.Private.CoreLib/src/System/Reflection/RuntimeAssembly.cs

```diff
@@ -722,9 +689,11 @@ public override Assembly GetSatelliteAssembly(CultureInfo culture, Version versi
             an.CultureInfo = culture;
             an.Name = name;
 
-            RuntimeAssembly retAssembly = nLoad(an, null, this, ref stackMark,
-                                IntPtr.Zero,
-                                throwOnFileNotFound);
+            // This stack crawl mark is never used in the native code sinde we specify the assembly
+            // explicitly.
+            StackCrawlMark stackMark = StackCrawlMark.LookForMe;
```

Comment:
I am not happy with this, but I don't see a straightforward way of not passing the stack mark to `nLoad` at all.

Reply:
Hmm, on second thought I can remove the parameter from `AssemblyNative::Load` and use `Assembly.GetExecutingAssembly(ref stackMark);` in `InternalLoadAssemblyName`.

Reply:
I commited the change to `AssemblyNative::Load`, but I am not sure of the possible performance implications.